Modules/Microsoft365DSC/DSCResources/MSFT_AADConditionalAccessPolicy/MSFT_AADConditionalAccessPolicy.psm1

```diff
@@ -941,10 +941,7 @@ function Set-TargetResource
         {
             $conditions.Applications.Add('IncludeApplications', $IncludeApplications)
         }
-        if ($ExcludeApplications)
-        {
-            $conditions.Applications.Add('ExcludeApplications', $ExcludeApplications)
-        }
+        $conditions.Applications.Add('ExcludeApplications', $ExcludeApplications)
```

Comment:
I just had a look at the issues description and I see a potential issue with this implementation. I someone would not specify the property, this code would replace the any existing configuration. In DSC we try to only modify the parameters that are set for the resource. In this instance, it would fail. Could you add additional testing, Like: Parameter is set and current values do not match the desired ones?

Reply:
done